frequency-chain/frequency-xyz
# feat: swap in Footer component from @frequency-chain/style-guide

- also fix tailwindow.config.js to use 'extend' property, else Tailwind classes from the package don't come through always


- Closes #36 

## Screenshot(s)

Desktop:
![image](https://github.com/user-attachments/assets/0cf2479b-0ab0-4dc9-a463-9858ed115259)

Mobile/small screen:
![image](https://github.com/user-attachments/assets/b6637fef-97be-422a-88cc-b00c59e00783)

Please delete options that are not relevant.

      re-approval.)

# How to Test?
1. Pull branch
2. Run `npm run dev -- --open`
3. Visually verify that the footer matches the screenshots
4. 
# Checklist:
